src/components/StatisticsCard/Battles.js

```diff
@@ -7,13 +7,13 @@ ChartJS.register(CategoryScale, LinearScale, BarElement, Title, Tooltip, Legend)
 const Battles = (props) => {
   const lastSnapshot = props.tankStats.data.snapshots.at(-1);
   const dataForTables = props.filteredStats.dataForTables;
-  const isDataEmpty = Object.keys(dataForTables).length > 0;
+  const isDataEmpty = Object.keys(dataForTables).length > 0
 
-  const months = new Date().getMonth() - new Date(props.tankStats.data.createdAt * 1000).getMonth() +
-    12 * (new Date().getFullYear() - new Date(props.tankStats.data.createdAt * 1000).getFullYear());
+  const months =  new Date().getMonth() - new Date(props.tankStats.data.createdAt * 1000).getMonth() +
+  12 * (new Date().getFullYear() - new Date(props.tankStats.data.createdAt * 1000).getFullYear());
```

Comment:
You can create variable `creationDate` to avoid duplicate code
```suggestion
  const creationDate = new Date(props.tankStats.data.createdAt * 1000);

  const months =  new Date().getMonth() - creationDate.getMonth() +
  12 * (new Date().getFullYear() - creationDate.getFullYear());

```